src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -34,7 +34,7 @@ const MyApp = () => {
 
 ## Constructor
 
-Accepts a config object of `logLevel`, `postLogConfig`, `simulcastEnabled`, `logger` and `videoDownlinkBandwidthPolicy` that will be used when initializing a meeting session. See [MeetingProvider](/docs/sdk-providers-meetingprovider--page) documentation for more details.
+Accepts a config object of `logLevel`, `postLogConfig`, `simulcastEnabled`, `enableWebAudio`, `logger` and `videoDownlinkBandwidthPolicy` that will be used when initializing a meeting session. See [MeetingProvider](/docs/sdk-providers-meetingprovider--page) documentation for more details.
```

Comment:
No change required, but we generally add new parameters at the end of existing parameter list.

Reply:
Since I put this `enableWebAudio` after `simulcastEnabled` in the meeting manager, so I do the same in the document.